app/Helpers/database/game.php

```diff
@@ -648,9 +649,14 @@ function modifyGameTitle(string $username, int $gameId, string $value): bool
         return false;
     }
 
-    $game->Title = $value;
+    $originalTitle = $game->title;
+
+    $game->title = $value;
     $game->save();
 
+    // Also, if necessary, update the game's sort title.
+    (new WriteGameSortTitleFromGameTitleAction())->execute($game, $originalTitle);
```

Comment:
Both of these should now be taken care of in latest.

I can't seem to find test coverage for the `submitgametitle` routine. It may be worth setting that up in a subsequent PR.